net-irc/znc/znc-1.6.1.ebuild

```diff
@@ -0,0 +1,198 @@
+# Copyright 1999-2015 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=5
+
+PYTHON_COMPAT=( python{3_3,3_4} )
+inherit base python-single-r1 systemd user
+
+MY_PV=${PV/_/-}
+GTEST_VER="1.7.0"
+GTEST_URL="https://googletest.googlecode.com/files/gtest-${GTEST_VER}.zip"
+DESCRIPTION="An advanced IRC Bouncer"
+
+if [[ ${PV} == "9999" ]]; then
```

Comment:
this is dead code/code duplication for no useful reason, drop all occurences of `[[ ${PV} == "9999" ]]` and related code